schematic/models/metadata.py

```diff
@@ -328,15 +331,15 @@ def submit_metadata_manifest(
                 )
 
                 logger.info(f"No validation errors occured during validation.")
-                return True
+                result = True
```

Comment:
@linglp since you are no longer returning here, python will execute the remainder of the function. It looks like this would  execute  associateMetadataWithFiles twice in some cases? We'd need to avoid that. If the function returns True here, and having the remainder of the code as you have it below, could work?

Reply:
@milen-sage thanks for pointing that out. I will have the function return a synapse ID here so that associateMetadataWithFiles won't get executed two times. The submit route requires validation of the manifest. If this part returns True (and we don't execute the remaining parts), we can't get synapse ID. I also changed the variable name from "result" to "syn_id" to avoid confusion.